data.sql

```diff
@@ -166,4 +166,80 @@ INSERT into animals (
 	17,
 	true,
 	3
-);
\ No newline at end of file
+);
+
+
+
```

Comment:
- [ ] It would be better if we perform the update operation on the animals' table and assign them owners like the examples below;
![Screenshot from 2021-12-23 17-01-40](https://user-images.githubusercontent.com/46388113/147271512-8379782b-5f2c-43d7-a867-6f84f05c4942.png)